Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM denoland/deno:latest
+FROM denoland/deno:1.19.0
 
 EXPOSE 8080
+EXPOSE 443
 
 WORKDIR /app
 
 USER deno
 
-ADD . .
+COPY allow_list.json ./allow_list.json
```

Comment:
As I understand it, it's best practice to not copy sensitive data across in the build process, better to pass them in at runtime. I tried to do this in the previous version but actually see that it wasn't doing what I wanted then anyway. I suggest removing this line and then moving `allow_list.json` across in the docker run command.

Reply:
Hey, I've made these changes and added a makefile to make running the command a bit easier (i.e. `make docker`). Shame about the add / copy thing being insecure, I think it means I can't reuse this dockerfile for deployment with Fly.io (as you don't get to call `docker run` yourself in that case).

I also updated replica-server to take an optional flag for the path of the allow list (which I use here).

Reply:
Maybe there would be a way to setup a volume for `allow_list.json` on fly.io?

Another approach could be to pass it in as an environment variable.

I don't really know _how_ insecure adding the file in the build is, someone would have to get into your server first to be able to check out your build images.... which is already a bad security breach...